### DISCUSSION TITLE
Rename "Purchase Contract" as "Specific Contract"

### DISCUSSION THREAD

**Discussion by paulakeen** [eprocurementontology/eprocurementontology#497] [Ideas]
The purchase contract could perhaps be renamed specific contract.

**Comment by paulakeen** [comment]
As far as "Purchase Contract" happens always in Framework Contract, then there should be no problem. Let's see it together in the WG.

**Comment by SellittoGiampaolo** [comment]
or "_contract deriving from a framework agreement_"
Actually some conditions that remain open in the FA may need a further negotiation leading to a specific contract 
also the term   _specific contract_  is convincing

**Comment by eprocurementontology** [comment]
The definition needs to be made as well

**Comment by JachymHercher** [comment]
I would suggest simply calling it "contract within a framework agreement" or "contract based on a framework agreement".  Other adjectives, such as "specific contract" would kind of imply that a "normal" public contract is not specific, which is strange. 

In additional information, we should have a list of synonyms, as I think there are many. OCDS may have quite a bit of experience with this - @jpmckinney?)

**Comment by jpmckinney** [comment]
@JachymHercher Do you mean experience with respect to synonyms? Since ePO has a separate glossary, I would put the synonyms in the glossary, and refer to glossary terms from the ontology.

I also agree that "contract within a framework agreement" or similar is much better than "purchase contract" or "specific contract".

**Comment by eprocurementontology** [comment]
FrameworkAgreementSpecificContract?

**Comment by JachymHercher** [comment]
@jpmckinney Sorry for not being clear. I meant a list of synonyms for "contract within a framework agreement" from different jurisdictions (e.g. "call-off contract"; "draw-down" contract in Ireland).

**Comment by jpmckinney** [comment]
Ah, yes, we can contribute that, though I don't have a list handy.

**Comment by andreea-pasare** [comment]
In latest ePO version, 3.1.0, the Contract is modeled in the core module as depicted in the diagram below: 
![image](https://user-images.githubusercontent.com/88081600/220904491-47e6cb96-3879-4792-b4e3-8525bca0ca6e.png)
